packages/screen-orientation/android/src/main/java/io/capawesome/capacitorjs/plugins/screenorientation/ScreenOrientation.java

```diff
@@ -75,13 +58,13 @@ public String getCurrentOrientationType() {
         int rotation = bridge.getActivity().getWindowManager().getDefaultDisplay().getRotation();
         switch (rotation) {
             case Surface.ROTATION_90:
-                return ScreenOrientationType.LANDSCAPE_PRIMARY;
+                return ScreenOrientationType.LANDSCAPE_PRIMARY.key();
```

Comment:
In this case you should also rename the method to `getCurrentOrientationTypeKey`.